app/javascript/app/components/table/table-styles.scss

```diff
@@ -26,6 +26,10 @@
     color: $theme-color;
     font-size: $font-size-sm;
 
+    div {
+      overflow: scroll;
```

Comment:
I cannot see this working. Please review the position of the content for Mitigation > Sectors covered. The flexbox positioning is rendering cenetered vertically without scroll.